project_iteration_sale_inheritance/__init__.py

```diff
@@ -0,0 +1,3 @@
+# © 2020 - today Numigi (tm) and all its contributors (https://bit.ly/numigiens)
```

Comment:
Codacy found an issue: [Missing docstring in public package (D104)](https://app.codacy.com/gh/Numigi/odoo-project-addons/pullRequest?prid=6223602)